packages/perseus/src/util/graphie.ts

```diff
@@ -1675,10 +1675,34 @@ const setLabelMargins = function (span: HTMLElement, size: Coord): void {
             marginTop: -height / 2 - y * scale,
         });
     } else {
+        const $container = $span.closest(".svg-image");
+        // Ensuring that the line-height of the text doesn't throw off placement of the text element.
+        // Inherited line-height values can really mess up placement.
+        $container.css("line-height", "normal");
+
+        // The 'max-width' value is the same as the expected (100% scale) width of the graphie.
+        // We are using jQuery to get this information since we don't have a way to pass it to this function.
+        const expectedWidth = $container.css("max-width") ?? "0px";
+        // We can calculate the true scale of the graphie by taking actual width and dividing by the expected width.
+        // NOTE: Using 'slice' to remove the "px" unit from the end of the expected width value.
```

Comment:
Would this value ever be in a unit other than `px`?

Reply:
Great question! I can't say for sure (I would need to dive into the code some more to find it). I think what is happening with `max-width` is it is taking the measurement from the image and slapping a "px" on it - that's my guess.

Reply:
I wonder if, instead of digging through, you could use an approach like this to only trim `px` if it exists: `expectedWidth.replace(/px$/, '');`?

Also, I suspect this is a nitpicky concern and likely is fine the way you have it. :)

Reply:
I like that! Thanks!!